finos/FDC3
# 165 Add FDC3 Explained to the toolbox folder

resolves #165 

I have recreated the FDC3 Explained pull request based on #367.

I have made the following changes:
- Rebased on latest master
- Move from website/static folder to toolbox/fdc3-explained
- Remove legacy version and demo apps
- Some renames and cleanup of files
- Improve handling of switching between different versions with drop-down
- Ensure that user informed no FDC3 version is detected when not running in a desktop agent
- Fix handling of OpenFin detection
- Use fdc3.getInfo for v1.2
- Add README.md
- Squashed commits to reduce churn

I believe if I did the build correctly, the deployment to the toolbox folder on the website should now also work, alongside that of FDC3 Workbench.

![image](https://user-images.githubusercontent.com/3295115/147417226-94ae1c71-665a-4a95-b7ac-6dc1ac8354fa.png)

@kriswest I believe this addresses all your issues:

- [x] Ensure all files encapsulated in a folder under the 'toolbox' folder to ensure we can use it for other tools, i.e. toolbox/fdc3-explained
- [x] Rename the root page from toolbox/fdc3_explained.html to toolbox/fdc3-explained/index.html (and correct relative URLs in that file).
- [x] Also move toolbox/fdc3_explained_logo.svg -> toolbox/fdc3-explained/logo.svg
- [x] Remove duplicate files (including legacy.html)
- [x] Ensure that the 1.2 version uses the the fdc3.getInfo() function to get provider details - meaning vendor specific code can be removed
- [x] Remove vendor specific manifest files in toolbox/openfin directory
- [x] Add a Readme file to the toolbox/fdc3-explained folder
